imap/itip_support.c

```diff
@@ -1016,6 +1016,12 @@ HIDDEN enum sched_deliver_outcome sched_deliver_local(const char *userid,
     syslog(LOG_DEBUG, "sched_deliver_local(%s, %s, %X)",
            sender, recipient, sparam->flags);
 
+    if (!strcmp(sender, recipient)) {
```

Comment:
This will skip nulling out txn just below on line 1023/1029, and then at the end we do:
```
done:
...
    spool_free_hdrcache(txn.req_hdrs);
...
```

Which I think might now call spool_free_hdrcache(...) on uninitialized data?

Reply:
Fixed